asf_tools/water_map.py

```diff
@@ -268,8 +279,9 @@ def make_water_map(out_raster: Union[str, Path], vv_raster: Union[str, Path], vh
         water_map = np.ma.masked_less_equal(gaussian_array, gaussian_threshold).mask
         water_map &= ~array.mask
 
-        write_cog(str(out_raster).replace('.tif', f'_{pol}_initial.tif'), water_map, transform=out_transform,
-                  epsg_code=out_epsg, dtype=gdal.GDT_Byte, nodata_value=False)
+        write_cog(str(out_raster).replace('.tif', f'_{pol}_initial.tif'),
+                  format_raster_data(water_map, padding_mask, nodata),
```

Comment:
In the definition of the function, I didn't see where you define the variable padding_mask, but here you use it. I don't know if it suitable. You may just format_raster_data(water_map, nodata=nodata). I found the same thing in three places.

Reply:
@cirrusasf the padding variable is created in [line #254](https://github.com/ASFHyP3/asf-tools/blob/format_raster_data/asf_tools/water_map.py#L254), so I just wanted to calculate that once for the entire water_map function, but wanted it to be redundant just in case it wasn't included in the `format_raster_data` function. I think I'd rather remove [Lines 135-137](https://github.com/ASFHyP3/asf-tools/blob/format_raster_data/asf_tools/water_map.py#L135-L137) if that would make more sense!

Reply:
Oh, I missed that part. I suggestion you keep Lines 135-137 as they are.